x-pack/plugins/enterprise_search/public/applications/shared/flash_messages/set_message_helpers.test.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  FlashMessagesLogic,
+  setSuccessMessage,
+  setErrorMessage,
+  setQueuedSuccessMessage,
+} from './';
+
+describe('Flash Message Helpers', () => {
+  const message = 'I am a message';
+
+  describe('setSuccessMessage()', () => {
+    it('sets a success message', () => {
+      FlashMessagesLogic.mount();
+      setSuccessMessage(message);
+
+      expect(FlashMessagesLogic.values.messages).toEqual([
+        {
+          message,
+          type: 'success',
+        },
+      ]);
+    });
+  });
+
+  describe('setErrorMessage()', () => {
+    it('sets an error message', () => {
+      FlashMessagesLogic.mount();
+      setErrorMessage(message);
+
+      expect(FlashMessagesLogic.values.messages).toEqual([
+        {
+          message,
+          type: 'error',
+        },
+      ]);
+    });
+  });
+
+  describe('setQueuedSuccessMessage()', () => {
+    it('sets a queued success message', () => {
+      FlashMessagesLogic.mount();
```

Comment:
Minor DRY suggestion so you don't have to repeat mount() before each test:

```suggestion
  beforeEach(() => {
    FlashMessagesLogic.mount();
  });

  describe('setSuccessMessage()', () => {
    it('sets a success message', () => {
      setSuccessMessage(message);

      expect(FlashMessagesLogic.values.messages).toEqual([
        {
          message,
          type: 'success',
        },
      ]);
    });
  });

  describe('setErrorMessage()', () => {
    it('sets an error message', () => {
      setErrorMessage(message);

      expect(FlashMessagesLogic.values.messages).toEqual([
        {
          message,
          type: 'error',
        },
      ]);
    });
  });

  describe('setQueuedSuccessMessage()', () => {
    it('sets a queued success message', () => {
```

Reply:
Another comment - do we really need a describe() block for a single it test? Can we simplify to just:

```js
it('setQueuedSuccessMessage()', () => {
  // just do the test
});
```

without the extra describes?

Reply:
Good catch! Done :)